sources/Common/src/ZipErrorMessage.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2025, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+#include "ZipErrorMessage.h"
+
+#include <sstream>
+
+
+std::string formatZipErrorMessage(const zip::ZipException& e) {
+  std::stringstream zipErrorStream;
```

Comment:
Why overriding the messages sent by libzip? The general exception catch should be enough ?

Reply:
The error message given by libzip is not always clear. When the zip::Error::LIBARCHIVE_INTERNAL_ERROR error code is thrown, the message is clear : ''Unrecognized archive format". But when the zip::Error::FILE_NOT_FOUND error code is thrown, only the name of the missing file is given. So the error message is not clear in that case.